### ISSUE TITLE
Pasted passwords disappear after interaction with GUI

### ISSUE THREAD

**Issue by godfuture** [marius-wieschollek/passwords#284]
Hi, entering new passwords is a hard job sometimes. I try to paste the new password into the password field, but when I continue with entering other fields, the password fields gets emptied. I noticed that when I manually add a character and delete it again the pasted string seems to be accepted and stays in the password field. For now it seems the field requires the user to really type the password.

The reason I want to paste the password is that I have generated the password outside the password app like random password generators or passwords predefined on devices.

Please take a look.

Edit:
System Information

    Nextcloud Version: 18.0.7
    PHP Version: 7.3.2
    Passwords Version: 2020.6.0
    Browser and Version: Any
    Client OS and Version: Any
    Server OS and Version: Any

Steps to reproduce

    Create new password
    Paste some string into password field
    Jump to other fields and fill them as usual

Actual result

    The password field gets emptied when typing in other fields.

Expected result

    The password field should not change.

**Comment by marius-wieschollek** [comment]
Please use the provided bug issue template when opening a new issue.

**Comment by marius-wieschollek** [comment]
I have not been able to reproduce this issue with Chromium and Firefox on Linux and Windows. Copying into the password field (right click + paste, Stgr+V, Mousewheel click on Linux) works correctly and jumping to any other field (with tab or by clicking it) does not clear the password field.

Can you add specific instructions to reproduce the error any maybe test this issue with a blank browser profile?

**Comment by godfuture** [comment]
Hi, with latest update it did not happen to me anymore. So lets close it. I will reopen, if it happens again.

**Comment by godfuture** [comment]
Hi, it just happened again. I was pasting from my password generator, and the password vanished once I modified the name or comment field.

Edit: the password was not pasted with context menu paste, but "insert latest password" https://addons.mozilla.org/de/firefox/addon/secure-password-generator/

The function "insert latest password" works everywhere just fine. As I experienced issues with vanished passwords also using context menu paste, I guess iti is a general issue in the nc password app.

v2020.8.0 (nc v18.0.9)
